### DISCUSSION TITLE
CMS Online Validator Tool Not Functioning Correctly?

### DISCUSSION THREAD

**Discussion by ibatis1213** [CMSgov/hospital-price-transparency#116] [Q&A] [answered]
Up until Thursday 6/13, I have successfully utilized the CMS Price Transparency Validator tool (https://cmsgov.github.io/hpt-tool/online-validator/) to verify the accuracy of our files. The check would return no errors. Since 6/13, all previously successfully tested files are now failing check and are showing the following error: 

![image](https://github.com/CMSgov/hospital-price-transparency/assets/173093339/93e34b8b-32c8-4614-9701-4e65ff406fd1)

Upon re-checking the requirements and the sample CSV template ( https://github.com/CMSgov/hospital-price-transparency/blob/master/documentation/CSV/templates/V2.0.0_Wide_CSV_Format_Template.csv) , the files appear to be compliant but still failing validator check.

Is this a known issue with the validator itself? Thank you for any input you may have.

**Comment by RidenourK** [comment, marked as answer]
@ibatis1213. Thank you for your question. 

Would you be willing to provide the first four rows of one of the MRFs the CMS HPT mailbox at [PriceTransparencyHospitalCharges@cms.hhs.gov](mailto:PriceTransparencyHospitalCharges@cms.hhs.gov) so we may investigate this further?

**Reply by ibatis1213** [reply]
I did just now, thank you very much, I appreciate the prompt response. Please let me know if anything else is needed.

**Comment by PTTCorroHealth** [comment]
Funny, as I just put a MRF through the validator completely missing any payer info and it passed w flying colors...?
